### PR TITLE
feat: update get-organisation-disruptions endpoint

### PR DESCRIPTION
**Pull request by DeannaSharma**

### CODE REVIEW

File: packages/organisations-api/get-organisation-disruptions/index.ts [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+};
+
+const getOrganisationDisruptions = async (orgId: string) => {
+    try {
+        const disruptionsTableName = process.env.DISRUPTIONS_TABLE_NAME as string;
+        const activeDisruptions = await getActiveDisruptions(disruptionsTableName, logger, orgId);
+
+        const disruptionsFormattedForMap = await Promise.all(
+            activeDisruptions.flatMap(async (disruption) => {
+                if (!disruption.consequences || disruption.consequences.length === 0) {
```
Comment (alainajaffer) [inline packages/organisations-api/get-organisation-disruptions/index.ts:27]: do we need to check disruption.consequences exists before doing disruption.consequences.length

File: packages/organisations-api/get-organisation-disruptions/index.ts [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+                };
+            }),
+        );
+
+        return disruptionsFormattedForMap.filter(notEmpty);
+    } catch (e) {
+        if (e instanceof Error) {
+            logger.error(`Error occurred while getting stops and services for an organisation: ${orgId} - error: `, e);
+
+            throw e;
```
Comment (alainajaffer) [inline packages/organisations-api/get-organisation-disruptions/index.ts:84]: do we need another throw e here

File: packages/organisations-api/get-organisation-disruptions/index.ts (hunk 1 above)
Comment (DeannaSharma) [reply packages/organisations-api/get-organisation-disruptions/index.ts:27]: i don't believe so

File: packages/organisations-api/get-organisation-disruptions/index.ts (hunk 2 above)
Comment (DeannaSharma) [reply packages/organisations-api/get-organisation-disruptions/index.ts]: removed line 84

**Review by alainajaffer — APPROVED** [review]

**Pull request merged**